src/components/Tags/Tags.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="tags">
+      <Tag v-for="tag in internalTags"
+           :key="tag"
+           v-bind="{icon, disabled, outlined, hovered, elevated, flat, color}"
+           class="mr-1"
+      >{{ tag }}</Tag>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+import Tag from "@/components/Tag/Tag.vue";
+export default defineComponent( {
+  name: "Tags",
+  components: {Tag},
+  props: {
+    icon: {
+      type: String,
+      default: null
+    },
+    tags: {
+      type: Array,
+      default: null
```

Comment:
This should probably default to an empty array since we're expecting it to be an array. Either than or remove the default and make it required. That might be better